core/services/relay/evm/functions/config_poller_test.go

```diff
@@ -205,3 +207,27 @@ func setFunctionsConfig(t *testing.T, pluginConfig *functionsConfig.ReportingPlu
 		OffchainConfig:        offchainConfig,
 	}
 }
+
+func generateDefaultOCR2OnchainConfig(t *testing.T, minValue *big.Int, maxValue *big.Int) []byte {
+	serializedConfig := make([]byte, 0)
```

Comment:
just stylistic, but I'm surprised the linter doesn't complain about this one:
```suggestion
	var serializedConfig []byte
```